rn-cli.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  getTransformModulePath() {
+    return require.resolve('react-native-typescript-transformer');
+  },
+  getSourceExts() {
+    return ['ts', 'tsx'];
+  },
+};
```

Comment:
For what is this file used?

Reply:
I had a problem with running application and adding this file helped me. But I've checked and it looks like now it works also without this file.